cfde_deriva/configs/portal/c2m2-level1-portal-model.json

```diff
@@ -2323,7 +2404,48 @@
             ],
             "missingValues": [ "" ],
             "primaryKey": "id"
-         }
+         },
+         "deriva": {
+            "source_definitions": {
+               "columns": true,
+               "fkeys": true,
+               "sources": {
+                  "S_biosample": {
+                     "markdown_name": "Biosample",
+                     "comment": "List of biosamples referencing this anatomy.",
```

Comment:
Rephrase as "this anatomical term"
(I initially put this comment on the wrong section)

Reply:
If you look a few more lines above, you used "Uberon term". Should we stay consistent and use "Uberon term" instead of "anatomical term"?

Reply:
Hmm, I default to the more generic concept.  Those references to specific external ontologies were introduced by other folks, and I worry they could be over-specific if our ontology working group eventually adopts terms from more than one source...